components/tikv_util/src/future_pool/builder.rs

```diff
@@ -105,12 +111,25 @@ impl Builder {
         } else {
             "future_pool"
         };
-        let env = Arc::new(super::Env {
-            on_tick: self.on_tick.take(),
+        let env = super::Env {
```

Comment:
Why `Arc` is not required anymore?

Reply:
I move `on_tick` in the original `Env` to the runner. Now `Env` only contains metrics, which are reference counted atomics themselves.